### PR TITLE
Messed up code probbly

### PR DESCRIPTION
**Pull request by ProFireDev**
ahhhhhhhhhhhhhhh

removed args and a bunch of stuff, also added some minor comments

♨♨🍝🍝🍝🍝🍝🍝🍝🍝

### CODE REVIEW

**Comment by ProFireDev** [comment]
that was pain but i think its stable? i mean the bot ran so uh, it must work.

File: config.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
         "invalidCmdReminder": true,
         "prefixReminder": true,
         "owner": [
-            "708333380525228082",
-            "125644326037487616"
+            "372390836694155264",
```
Comment (emilianya) [inline config.json:8]: Oi this needs to change back to what it was

File: config.json [hunk 2]
```diff
@@ -29,9 +29,7 @@
         "dmOwner": true
     },
     "reports": {
-        "enabled": true,
-        "channelName": "reports",
-        "staffRoleName": "Staff"
```
Comment (emilianya) [inline config.json:34]: Why were these removed

**Review by emilianya — CHANGES_REQUESTED** [review]
See comments

File: index.js [hunk 3]
```diff
@@ -238,7 +240,8 @@ client.on("message", message => {
 	}
 });
 
-client.on("messageUpdate", (oldMessage, newMessage) => {
```
Comment (emilianya) [inline index.js:283]: did you make sure this works? looks like something where it gives you back 2 values and its the old and new, if thats the case this would make it think the new is just the same no?

File: index.js [hunk 4]
```diff
@@ -5,10 +5,10 @@ const Discord = require("discord.js");
...
 const client = new Discord.Client({partials: ["MESSAGE", "CHANNEL", "REACTION", "USER"]});
 const chalk = require("chalk");
-const success = chalk.green;
-const warn = chalk.yellow;
-const error = chalk.bold.red;
-const info = chalk.blue;
+//const success = chalk.green;
+//const warn = chalk.yellow;    //all of thease seem to do nothing, why
+//const error = chalk.bold.red;
+//const info = chalk.blue;
```
Comment (nyakase) [inline index.js:11]: They were added for future usage, but I'm not entirely sure if we need them

File: package.json [hunk 5]
```diff
@@ -10,15 +10,15 @@
...
     "mysql": "^2.18.1",
     "ora": "^5.3.0",
     "twitter": "^1.7.1"
   },
   "devDependencies": {
-    "eslint": "^7.17.0",
+    "eslint": "^7.22.0",
     "inquirer": "^8.0.0",
-    "npm": "^7.4.1"
+    "npm": "^7.6.3"
```
Comment (Vukky123) [inline package.json:21]: You seem to have updated some dependencies here! Although that's nice of you to do, we use Dependabot for this, so keep that in mind in the future.

**Comment by ProFireDev** [comment]
marked as closed in favor for a new edited larger commit

**Comment by ProFireDev** [comment]
ill edit this and reopen it later probably
